vgen/src/main/resources/io/gapi/vgen/common/common.snip

```diff
@@ -1,4 +1,3 @@
-# TODO(jgeiger): move to non-Python-specific location.
```

Comment:
Can we remove the "common" directory for shared snippets, and just keep them one level higher on the directory tree?